bootstrap/exherbo.sh

```diff
@@ -0,0 +1 @@
+_exherbo_common.sh
```

Comment:
I've made the requested changes.

For Gentoo however it is more likely that the _gentoo_common.sh will be required in the near future since there exist some derivatives like e.g. Funtoo and Sabayon.